WordPress/src/main/res/values/strings.xml

```diff
@@ -192,7 +192,8 @@
     <string name="theme_auth_error_message">Ensure you have the privilege to set themes</string>
     <string name="theme_current_theme">Current theme</string>
     <string name="theme_premium_theme">Premium theme</string>
-    <string name="theme_no_search_result_found">No results found</string>
+    <string name="theme_no_search_result_found">Sorry, no themes found.</string>
```

Comment:
Can we drop the `Sorry` and the final period? (for consistency + https://www.google.com/design/spec/style/writing.html#writing-capitalization-punctuation)